tensorflow/examples/android/src/org/tensorflow/demo/CameraActivity.java

```diff
@@ -213,6 +387,7 @@ public boolean onKeyDown(final int keyCode, final KeyEvent event) {
     return super.onKeyDown(keyCode, event);
   }
 
+  protected abstract void processImageRGBbytes(byte[] luminance) ;
```

Comment:
Ok, I think I understand the intention now. Can you please remove luminance (I'll add it back later), and make rgbBytes private and pass it as the parameter to the processRGBBytes() method?